### PR TITLE
increase wait time in TestProjectionParallelRequiredRows to fix CI problem

### PR DESCRIPTION
**Pull request by qw4990**
### What problem does this PR solve? 
CI problem caused by background goroutines in `TestProjectionParallelRequiredRows`.

### What is changed and how it works?
Try to increase wait time in TestProjectionParallelRequiredRows to reduce the probability of this problem.

### Check List 

Tests 

 - Unit test

### CODE REVIEW

File: executor/executor_required_rows_test.go [hunk 1]
```diff
@@ -575,7 +575,7 @@ func (s *testExecSuite) TestProjectionParallelRequiredRows(c *C) {
 
 			// wait projectionInputFetcher blocked on fetching data
 			// from child in the background.
-			time.Sleep(time.Millisecond * 5)
+			time.Sleep(time.Millisecond * 25)
```
Comment (eurekaka) [inline executor/executor_required_rows_test.go]: Do we have a deterministic approach to do the synchronization?

Comment (qw4990) [reply executor/executor_required_rows_test.go]: No.. we can't trace states of these background goroutines now 😢..
And it's ugly to introduce this feature just for this unit test..

**Comment by qw4990** [comment]
I ran all test 5 times and this problem didn't appear.
It seems ok now? @eurekaka

**Pull request merged**